index.html

```diff
@@ -105,31 +105,162 @@ <h2>Introduction</h2>
+          <section>
+            <h2>Credential Header</h2>
+            <aside
+              class="example"
+              title="verifiable credential header (decoded JWT header)"
+            >
+              <pre data-include="./examples/vc-2.0/templates/vc+ld+jwt/header.json" data-include-format="json"></pre>
+            </aside>
+          </section>
+          <section>
+            <h2>Credential</h2>
+            <aside
+              class="example"
+              title="verifiable credential (decoded JWT claimset)"
+            >
+              <pre data-include="./examples/vc-2.0/templates/vc+ld+jwt/claimset.json" data-include-format="json"></pre>
+            </aside>
+          </section>
+          <section>
+            <h2>Verifiable Credential</h2>
+            <aside
+              class="example"
+              title="verifiable credential (encoded as JWT)"
+            >
+              <pre data-include="./examples/vc-2.0/vc+ld+jwt.jose" data-include-format="json"></pre>
+            </aside>
+          </section>
+        </section>
+        <section>
+          <h2>Securing <code>application/vp+ld+json</code> with JOSE</h2>
+          <p>[[rfc7515]] MAY be used to secure this media type.</p>
+          <p><code>typ</code> MUST be <code>vp+ld+jwt</code></p>
+          <p><code>cty</code> MUST be <code>vp+ld+json</code></p>
+          <p>See <a data-cite="rfc7515#section-4.1.10">Common JOSE Header Parameters</a> for additional details.</p>
+          <section>
+            <h2>Presentation Header</h2>
+            <aside
+              class="example"
+              title="verifiable presentation header (decoded JWT header)"
+            >
+              <pre data-include="./examples/vc-2.0/templates/vp+ld+jwt/header.json" data-include-format="json"></pre>
+            </aside>
+          </section>
+          <section>
+            <h2>Presentation</h2>
+            <aside
+              class="example"
+              title="verifiable presentation (decoded JWT claimset)"
+            >
+              <pre data-include="./examples/vc-2.0/templates/vp+ld+jwt/claimset.json" data-include-format="json"></pre>
+            </aside>
+          </section>
+          <section>
+            <h2>Verifiable Presentation</h2>
+            <aside
+              class="example"
+              title="verifiable presentation"
+            >
+              <p>An encoded verifiable presentation (using external proof).</p>
+              <pre data-include="./examples/vc-2.0/vp+ld+jwt.jose" data-include-format="text"></pre>
+            </aside>
+          </section>
+        </section>
+      </section>
+      <section>
+        <h2>With COSE</h2>
+        <section>
+          <h2>Securing <code>application/vc+ld+json</code> with COSE</h2>
+          <p>[[rfc8152]] MAY be used to secure this media type.</p>
+          <p><code>type (TBD)</code> MUST be <code>vc+ld+cwt</code></p>
```

Comment:
This is the VC-JWT specification, not the VC-CWT specification. This seems to be a change in scope for this specification, with technologies that are not listed in our charter.

How are we going to defend these additions, of COSE and CWT, to the Director and the W3C Membership when our charter doesn't mention these technologies at all?

Reply:
In practice, once `vc+ld+json` is registered we should expect it to start showing up in all the places where `cty` is allowed.

I think it is better to address the alignment between `jwt` and `cwt` secured `vc+ld+json` up front, to avoid the problems we saw in the previous work, where the vc-jwt format was designed without a broader context.

The same "key resolution" problems will occur for COSE and JOSE, and they can both be used to secure `application/vc+ld+json`, if we don't comment on it, people will just do it... many different ways.

The day 3 resolution also allows for this to be addressed outside the working group with a mapping from vanilla CWT to JSON-LD.

I have not attempted to define that mapping, but perhaps we should for consistency?